operating/manage-compliance-operator/compliance-operator-rhacs.adoc

```diff
@@ -32,4 +32,4 @@ $ oc -n stackrox delete pod -lapp=sensor
 After performing these steps, run a compliance scan in {product-title-short} and ensure that `ocp4-cis` and `ocp4-cis-node` results are displayed. See "Running a compliance scan" in the "Additional resources" section for more information.
 
 .Additional resources
-* xref:../../operating/manage-compliance.adoc#run-compliance-scan_manage-compliance[Running a compliance scan in {product-title-short}]
\ No newline at end of file
+* xref:../../operating/manage-compliance/managing-compliance-10.adoc#run-compliance-scan_managing-compliance-10[Running a compliance scan in {product-title-short}]
```

Comment:
@dashrews78 WDYT?

Reply:
I'm not 100% tracking the question here.  Above we have the important note that they don't have to use these object directly.  Now I could see an argument to put a competing link to 2.0 in the Additional resources section.

```
* If you are using Compliance (2.0), you can schedule your scan by using {product-title-short} to create a compliance scan schedule instead of creating the `ScanSettingBinding` on the Compliance Operator.
+
For more information about scheduling a compliance scan by using Compliance (2.0), see "Creating a compliance scan schedule" in the "Managing compliance (2.0)" section of the _Operating guide_.
* This example uses `ocp4-cis` and `ocp4-cis-node` profiles, but {ocp} provides additional profiles. See "Understanding the Compliance Operator" in the "Additional resources" section for more information.
```

Reply:
I think it has to be in the integration section as that consolidates the interaction with CO.  If we moved it to the Compliance v1.0 section, I think we would have to nest a configure CO page under there instead of going to the dashboards.   Now I suppose we could do something like `Using Compliance Operator with RHACS 2.0` or something.  Or even make a Compliance V1 section that houses all CO and manage compliance instructions and do the same thing for Compliance V2, though that may introduce some duplication.

Reply:
I'm ok with the note on top of the binding then so users are presented with something.